pymc3/distributions/continuous.py

```diff
@@ -640,9 +639,8 @@ def __init__(
     ):
         if sd is not None:
             sigma = sd
-            warnings.warn("sd is deprecated, use sigma instead", DeprecationWarning)
         tau, sigma = get_tau_sigma(tau=tau, sigma=sigma)
-        self.sigma = self.sd = tt.as_tensor_variable(sigma)
```

Comment:
I think we can leave `self.sd` too.

Reply:
Sure, I can revert this, but wouldn't it be safer to have a single source of truth for the value of sigma?

Reply:
I agree with your sentiment, but from user perspective, setting `sd=x` I would expect there to be a `y.sd`, and if we're keeping it around, which isn't the cleanest anyway, I don't see what we gain by changing this one thing in a subtle way.

Reply:
makes sense, thanks for explaining!